admin/lib/generators/solidus_admin/install/install_generator.rb

```diff
@@ -4,7 +4,11 @@ module SolidusAdmin
   module Generators
     class InstallGenerator < Rails::Generators::Base
       def install_solidus_core_support
-        route "mount SolidusAdmin::Engine, at: '/solidus_admin'"
+        route <<~RUBY
+          mount SolidusAdmin::Engine, at: '/admin', constraints: ->(req) {
```

Comment:
That's exactly the idea I had in mind. Happy you tried it out :slightly_smiling_face: